### ISSUE TITLE
cursorWordPartStartLeft should handle all caps names

### ISSUE THREAD

**Issue by wcscmp** [Microsoft/vscode#53938]
In a all caps symbols (e.g. FILE_SHARE_READ, FILE_ATTRIBUTE_NORMAL) cursorWordPartStartLeft skips only one letter but it should skip to underscore.
It would also be nice for it to able to handle all caps abbreviation in camel case text e.g. IDGenerator.

**Comment by martinschayna** [comment]
Also, multiple spaces should be skipped at once, typically indentation spaces.

**Comment by briman0094** [comment]
I agree; I was just about to report the same issue when I noticed this one. Same with `cursorWordPartRight` of course, which leads to the second thing I was about to report before noticing this issue:

The "camel case left" actions are `cursorWordPartStartLeft` while "camel case right" are `cursorWordPartRight`. I'm guessing the word "Start" is not intended to be in `cursorWordPartStartLeft`.
